clodius/multivec.py

```diff
@@ -41,27 +44,31 @@ def bedfile_to_multivec(input_filenames, f_out,
     print('base_resolution:', base_resolution)
     warned = False
 
-    for lines in zip(*files):
+    for count, lines in enumerate(zip(*files)):
```

Comment:
You don't use `count`... were you intending to include it in an error message somewhere?

Reply:
Or for debugging that got taken out. Replacing it with `_` for now.